Lib/fontbakery/specifications/ufo_sources_test.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import defcon
+import pytest
+
+from fontbakery.checkrunner import (DEBUG, ENDCHECK, ERROR, FAIL, INFO, PASS,
```

Comment:
Remove ENDCHECK here. Travis complains because this was not used at all.

At https://travis-ci.org/googlefonts/fontbakery/builds/355967025 you see:
```
************* Module fontbakery.specifications.ufo_sources_test
W:  8, 0: Unused ENDCHECK imported from fontbakery.checkrunner (unused-import)
```